components/vertex-components/pyproject.toml

```diff
@@ -1,32 +1,30 @@
-[project]
+[tool.poetry]
 name = "vertex-components"
+version = "0.1.0"
 authors = [
     {name = "Example User", email = "user@example.com"},
 ]
-description = "Vertex AI components"
+description = "Vertex components for training and prediction using the Chicago Taxi Dataset"
```

Comment:
these components aren't specific to the usecase - maybe description should be something like "KFP components for interacting with Vertex AI"